packages/design-system/src/components/Anchor/index.tsx

```diff
@@ -16,17 +18,15 @@ export const Anchor: React.FC<AnchorProps> = ({
   };
 
   return (
-    <a
+    <span
       className={classNames(
         "text-sm",
         decorationVariants[decoration],
         className
       )}
-      href={link}
-      role={role}
     >
-      {text}
-    </a>
+      <LinkTag href={href}>{children}</LinkTag>
```

Comment:
shouldn't we simply do sth like this:
```suggestion
      {children}
```